bin/MadGraph5_aMCatNLO/cards/ggh01_Toa01a01_Tomumutautau/haa_h125_10_gp/ggh01_M125_Toa01a01_M10_Tomumutautau_proc_card.dat

```diff
@@ -0,0 +1,17 @@
+set loop_color_flows False
+set max_npoint_for_channel 0
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set gauge unitary
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```

Comment:
this is default or? if so, please remove. see again my slides.

Reply:
is the model per se 4 fs or 5fs ?

Reply:
Checked it myself, looks like 4f

Reply:
The lines are default it seems. I will remove them in the next commit.